tests/Type/EnumTypeNativeEnumTest.php

```diff
@@ -0,0 +1,69 @@
+<?php declare(strict_types=1);
+
+namespace GraphQL\Tests\Type;
+
+use GraphQL\GraphQL;
+use GraphQL\Tests\Type\PhpEnumType\StringPhpEnum;
+use GraphQL\Type\Definition\EnumType;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Schema;
+use PHPUnit\Framework\TestCase;
+
+final class EnumTypeNativeEnumTest extends TestCase
```

Comment:
Please integrate this into `EnumTypeTest`

Reply:
Done @spawnia